.ci/azure/linux.yml

```diff
@@ -1,84 +1,101 @@
-    workingDirectory: $(WORK_DIR)
-    displayName: 'Install dependencies'
-
-  - task: CMake@1
-    inputs:
-      cmakeArgs: -GNinja -DVERBOSE_BUILD=ON -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) -DENABLE_JAVA=OFF -DIE_EXTRA_MODULES=$(REPO_DIR)/modules $(OPENVINO_REPO_DIR)
-      workingDirectory: $(BUILD_DIR)
-
-  - script: ninja
-    workingDirectory: $(BUILD_DIR)
-    displayName: 'Build Lin'
-
-  - script: ls -alR $(OPENVINO_REPO_DIR)/bin/
-    displayName: 'List files'
+resources:
+  repositories:
+  - repository: openvino
+    type: github
+    endpoint: openvinotoolkit
+    name: openvinotoolkit/openvino
+
+  - repository: testdata
+    type: github
+    endpoint: openvinotoolkit
+    name: openvinotoolkit/testdata
+
+jobs:
+- job: Lin
+  # About 150% of total time
+  timeoutInMinutes: 60
+
+  pool:
+    name: LIN_VMSS_VENV_F8S_WU2
+
+  variables:
+    system.debug: true
+    VSTS_HTTP_RETRY: 5
+    VSTS_HTTP_TIMEOUT: 200
+    WORKERS_NUMBER: 8
+    BUILD_TYPE: Release
+    REPO_DIR: $(Build.Repository.LocalPath)
+    OPENVINO_REPO_DIR: $(REPO_DIR)/../openvino
+    MODELS_PATH: $(REPO_DIR)/../testdata
+    WORK_DIR: $(Pipeline.Workspace)/_w
+    BUILD_DIR: $(WORK_DIR)/build
+    BIN_DIR: $(OPENVINO_REPO_DIR)/bin/intel64/$(BUILD_TYPE)
+
+  steps:
+  - script: |
+      curl -H Metadata:true --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2019-06-01"
+      whoami
+      uname -a
+      which python3
+      python3 --version
+      which java
+      java -version
+      gcc --version
+      lsb_release
+      env
+      cat /proc/cpuinfo
+      cat /proc/meminfo
+      cat /etc/fstab
+      vmstat -s
+      df
+      lsblk -o NAME,HCTL,SIZE,MOUNTPOINT | grep -i "sd"
+    displayName: 'System info'
+
+  - script: |
+      rm -rf $(WORK_DIR) ; mkdir $(WORK_DIR)
+      rm -rf $(BUILD_DIR) ; mkdir $(BUILD_DIR)
+    displayName: 'Make dir'
+
+  - checkout: self
+    clean: true
+    lfs: false
+    submodules: recursive
+    path: openvino_contrib
+
+  - checkout: openvino
+    clean: true
+    lfs: false
+    submodules: recursive
+    path: openvino
+
+  - checkout: testdata
+    clean: true
+    lfs: false
+    path: testdata
+
+  - script: |
+      sudo apt --assume-yes install libusb-1.0-0-dev
+      # Speed up build
+      wget https://github.com/ninja-build/ninja/releases/download/v1.10.0/ninja-linux.zip
+      unzip ninja-linux.zip
+      sudo cp -v ninja /usr/local/bin/
+    workingDirectory: $(WORK_DIR)
+    displayName: 'Install dependencies'
+
+  - task: CMake@1
+    inputs:
+      cmakeArgs: -GNinja -DVERBOSE_BUILD=ON -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) -DENABLE_JAVA=OFF -DENABLE_TESTS=ON -DIE_EXTRA_MODULES=$(REPO_DIR)/modules $(OPENVINO_REPO_DIR)
+      workingDirectory: $(BUILD_DIR)
+
+  - script: ninja
+    workingDirectory: $(BUILD_DIR)
+    displayName: 'Build Lin'
+
+  - script: ls -alR $(OPENVINO_REPO_DIR)/bin/
+    displayName: 'List files'
+
+  - script: java -cp ".:*" OpenVinoTestRunner
```

Comment:
test failed but CI is green...

Reply:
@azhogov , @ilya-lavrenov , do you know the reason why CI is passed with failed tests?

Reply:
Never mind - seems like it's internal Java tests issue.